src/js/components/date-picker.js

```diff
@@ -877,7 +876,8 @@ const enhanceDatePicker = (el) => {
     "usa-sr-only",
     DATE_PICKER_INTERNAL_INPUT_CLASS
   );
-  internalInputEl.id = "";
+  internalInputEl.removeAttribute("id");
+  internalInputEl.removeAttribute("name");
```

Comment:
Not breaking, but was leaving an empty name attribute. Went ahead and removed it.